src/components/NewOrder.js

```diff
@@ -1,26 +1,76 @@
 import { Component, DOM as dom } from 'react';
 import { connect } from 'react-redux';
 import * as actions from '../action_creators';
+import _ from 'underscore';
 
 const mapStateToProps = function (state) {
   return {
-    orders: state.orders,
     items: state.items
   };
 };
 
 class NewOrder extends Component {
+  constructor(props) {
+    super(props);
+    this.selectItem = this.selectItem.bind(this);
+    this.addOrderItem = this.addOrderItem.bind(this);
+    this.state = {
+      items: []
+    };
+  }
+
+  componentWillMount() {
+    this.props.loadItems();
+  }
+
+  addOrderItem() {
+    const defaultValue = this.props.items[0];
+    const items = this.state.items.concat(this.state.selectedItem || defaultValue);
+
+    this.setState({ items });
+  }
+
+  selectItem(e) {
+    const itemId = e.target.value;
+    const selectedItem = _.find(this.props.items, (item) => item.id === itemId);
+    this.state.selectedItem = selectedItem;
+  }
+
   render() {
     return dom.div(
       null,
       dom.h1(null, 'New Order Page'),
-      dom.button({
-          className: 'btn btn-default',
-          onClick: this.props.addOrder
-        }, 'Submit')
+      dom.select(
+        { id: 'item', onChange: this.selectItem },
+        this.props.items.map((item) => {
```

Comment:
I guess doesn't really matter, but does it not complain about setting `key` for iterated children?